src/v3/src/transformer/terminal/transformTerminalTransaction.ts

```diff
@@ -77,13 +78,15 @@ const appendViewLinks = (
   uischema: UISchemaLayout,
   widgetProps: WidgetProps,
 ): void => {
-  const { baseUrl } = widgetProps;
+  const { baseUrl, features } = widgetProps;
+  const isCancelAvailable = shouldShowCancelLink(features);
   const cancelStep = transaction?.availableSteps?.find(({ name }) => name === 'cancel');
   const cancelLink: LinkElement = {
     type: 'Link',
     options: {
       label: loc('goback', 'login'),
       step: 'cancel',
+      isActionStep: true,
```

Comment:
This is the cruz of the fix, since this was not being set, we were sending the request as a step instead of as an action, and auth-js failed when clicking.